src/Framework/PaymentGateways/Routes/RouteSignature.php

```diff
@@ -38,12 +60,43 @@ public function toString()
     }
 
     /**
-     * @since 2.19.0
+     * @unreleased
+     *
+     * @return string
+     */
+    public function toHash()
+    {
+        return wp_hash($this->signature);
+    }
+
+    /**
+     * Create expiration timestamp
+     *
+     * @unreleased
+     *
+     * @return string
+     */
+    public static function createExpirationTimestamp()
```

Comment:
@jonwaldstein Is there any reason to make this function static because it mostly belongs to this class and will be used internally.

Reply:
@ravinderk Jon is using the static function in his tests to control the timestamp used by the class.

Reply:
yeah I just did that for the tests but i'll make this non-static, it's so simple.

Reply:
resolve beafce60